scripts/ci/test_state.sh

```diff
@@ -5,3 +5,18 @@
 export STATE_IMAGES_AVAILABLE="${SHARED_DIR:-/tmp}/stackrox_ci_state_images_available"
 export STATE_BUILD_RESULTS="${SHARED_DIR:-/tmp}/stackrox_ci_state_build_results"
 export STATE_DEPLOYED="${SHARED_DIR:-/tmp}/stackrox_ci_state_deployed"
+
+# For the upgrade test
+export UPGRADE_PROGRESS_SENSOR_BUNDLE="${SHARED_DIR:-/tmp}/stackrox_ci_state_upgrade_progress_sensor_bundle"
```

Comment:
How about extracting `"${SHARED_DIR:-/tmp}/stackrox_ci_state_upgrade_progress_` to a variable?

Reply:
done.